### DISCUSSION TITLE
Port3 Network and Akash Network  PR collabs Discussion

### DISCUSSION THREAD

**Discussion by Jasper8321** [akash-network/community#540] [General]
Hello，Akash Network.
I'm mod Jasper from the community of Port3Network. We've been following your project for a long time. We know that you're focusing on developing in the AICrypto field. Port3Network is an AI service protocol. As a project that is also developing in the AI field, we would like to reach a friendly cooperation with your project:
     Port3Network is led by Jump Crypto and Kucoin Ventures, utilizes AI to index and standardize web3's decentralized data network. Port3Network has AI service protocols, Decalc (Decentralized Calculation) network, OpenBQL, SoPad and other infrastructures. Port3 has more than 70,000 DAUs and more than 4.8 million total users distributed across more than 500,000 mobile devices worldwide.
     Port3Network is developing a comprehensive solution on EVM and Solana for any dapp to train and enable and monetize AI models.

      We sincerely hope that Port3 will become an ecosystem partner for your platform, utilizing our expertise to develop innovative AI solutions to promote the AICrypto space. We'd love to work with you on PR!

Looking forward to your reply.
Sincerely,
A collection of Port3 related links:
Official Website: https://www.port3.io/
Twitter: https://twitter.com/Port3Network
Announcements: https://t.me/port3network
Discord Community: https://discord.com/invite/JV5gT5cGCk
Telegram Community: https://t.me/Port3_Network
OpenBQL: https://openbql.org/
SoPad: https://sopad.xyz/

**Comment by brewsterdrinkwater** [comment]
Hello. Thank you for creating the discussion. 

Akash is open source, so I will ask some questions, and I am sure others will chime in as well.

1. How do you envision "cooperating" with the Akash Network?
2. Have you or your technical team deployed anything on the Akash Network?
3. Does Port3 leverage compute resources for training or inference of AI Models? What kind of resources are you currently using?
4. Do you all "cooperate" with any projects in Cosmos?

I am looking forward to learning more.

**Reply by Caroleeeee** [reply]
Hello there, here's Carole, the growth manager from Port3. Here's our reply to the 5 questions above, plz feel free to check, and looking forward to discussing the collab details via emails/groups soon.

1/ How do you envision "cooperating" with the Akash Network?

Port3 Network is an AI Servicing Layer, led by Jump Crypto. Port3 Construct a network for AI and intelligent strategy services with massive Web3 datasets, decentralized computing platforms, and an open cross-chain execution layer.

There are four main infrastructures:
- AI Servicing Layer: Indexing and standardizing a Decentralized Data Network for Web3 using AI
- DeCalc: Fueling Web3's Growth with Federated Learning
- OpenBQL: AI-powered layer for efficient cross-chain interactions
- SoPad: Launching tomorrow trustless networks

**For cooperation between Port3 and Akash:**
1. We recognize that we have many data sources that need to be handled. We can utilize Akash's decentralized compute network to aggregate and standardize data collection. By assigning tasks to the provider, we get results from the Akash network.
2. You guys support deploying frontend pages, and we would love to deploy our static page on the Akash Network. And a joint press release announcing/AMAs/collab events could be a great first step to showcasing the power of our representation.

2/ Have you or your technical team deployed anything on the Akash Network?

Not yet, but we are researching integrating into Akash Network, and we are interested in cooperating, Port3 can deploy an application with Akash cloud service. (More details can be settled down via our collab talk)

3/ Does Port3 leverage compute resources for training or inference of AI Models?

Yes, Port3 is planning to leverage compute resources for both training and inference of AI models to aid in translating natural language to BQL scripts.

Port3 has developed an AI Intent Oriented blockchain interaction language, BQL. OpenBQL serves as an AI-enabled automation layer, facilitating seamless network interoperability across various blockchain ecosystems such as EVM, BTC, Solana, and more. OpenBQL enables intelligent cross-chain interactions and continuous functionality through smart contracts and an intermediate operations layer.

We plan to build an assistant that helps translate natural language to BQL scripts. For this, we need to train our NLP model to map between them, to handle on-chain interactions.

4/ What kind of resources are you currently using?
We have deployed a test page using Akash Network’s Deployment. 

5/ Do you "cooperate" with any projects in Cosmos?
Yes, we do, We've integrated with Sei Network and made a partnership shout-out with their ecosystem.
https://twitter.com/Port3Network/status/1696440235830530434

**Comment by Jasper8321** [comment]
Port3's four major infrastructures

⚫️AI Service Protocol: indexing and standardising Web3's decentralised data network using AI.
⚫️DeCalc: providing computational support for the data layer to drive Web3 development
⚫️OpenBQL: AI-powered efficient cross-chain interaction layer
⚫️SoPad: monetising social influence with Launchpad

Recently we've been working with AIOZ, Solana, PublicAI_, Jupiter, all of them!，Take a look at our recent our collaborations and events, here's our Weekly Report from last week: https://medium.com/@Port3/port3-weekly-report-april-1st-april-7th-13f3df2d2d91

We are mainly looking for a PR partnership with Akash, partner promotion on our official tweets, but also other partnership expansion, I can invite our BD manager to discuss with you in detail!

Sincerely look forward to your reply
